jupyterhub/tests/test_singleuser.py

```diff
@@ -167,6 +173,10 @@ async def test_disable_user_config(request, app, tmpdir, full_spawn):
     )
     assert r.status_code == 200
 
+    if IS_JUPYVERSE:
+        # the remaining is jupyter-server specific
+        return
```

Comment:
Rather than returning, this should verify that no jupyverse config files are loaded from $HOME.

Reply:
I still think this is jupyter-server specific. Jupyverse uses the [Asphalt application framework](https://github.com/asphalt-framework/asphalt) where [configuration files](https://asphalt.readthedocs.io/en/latest/userguide/deployment.html) are passed explicitly, so there is no notion of looking up directories for configuration files.

Reply:
Ah, then I'd just update the comment to say that jupyverse never loads any configuration from the home directory, and the skip can move to the top of the test, since there's nothing to check for jupyverse.